cli/main.cpp

```diff
@@ -45,14 +44,14 @@ int main(int argc, char** argv) {
         auto cli = lyra::cli()
             | lyra::help(show_help)
             | lyra::opt(show_version             )["-v"]["--version"           ]("Display version info and exit.")
-            | lyra::opt(list_dialect_paths       )["-l"]["--list-dialect-paths"]("List search paths in order and exit.")
+            | lyra::opt(list_search_paths        )["-l"]["--list-search-paths" ]("List search paths in order and exit.")
             | lyra::opt(clang,          "clang"  )["-c"]["--clang"             ]("Path to clang executable (default: '" THORIN_WHICH " clang').")
-            | lyra::opt(plugins,        "dialect")["-d"]["--dialect"           ]("Dynamically load dialect [WIP].")
-            | lyra::opt(dialect_paths,  "path"   )["-D"]["--dialect-path"      ]("Path to search dialects in.")
+            | lyra::opt(plugins,        "dialect")["-d"]["--dialect"           ]("Dynamically load plugin.")
```

Comment:
If dialects are now called plugins, the option should probably be
`-p` `--plugin`
and
`-P` `--plugin-path`.

Reply:
yes. Also thought about this, but for now I didn't want to touch each and every test case